configuration/boot_folder.md

```diff
@@ -43,6 +43,14 @@ When this file is present, SSH will be enabled on boot. The contents don't matte
 
 This is the file to configure wireless network settings (if the hardware is capable of it). Edit the country code and the network part to fit your case. More information on how to use this file can be found in [the `wireless/headless` section](./wireless/headless.md).
 
+### run_once
+
+If a `/boot/run_once` executable exists at boot time, Raspberry Pi OS executes it on the first boot. It may contain bash, python, or any executable ARM binary. After execution, `run_once` will be renamed to `run_once.XXX`. Later reboots will know `run_once.XXX` was already executed.
```

Comment:
I guess technically it's "executes it on the next boot" rather than just "first boot" ? (I think there's nothing which means it _only_ runs programs on first boot?)